README.md

```diff
@@ -61,180 +23,109 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 HTML/CSS & JavaScript capstone project <a name="about-project"></a>
+This capstone project build is based on an online website for a conference. 
+This capstone project aims to create a robust and user-friendly online conference website that revolutionizes the way conferences are conducted, providing a dynamic and engaging platform for attendees to connect, learn, and collaborate.
+>
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="#">HTML</a></li>
+     <li><a href="#">CSS</a></li>
+      <li><a href="#">JavaScript</a></li>
   </ul>
 </details>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- [ ] **Home page**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p
 
-> Add a link to your deployed project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- [Live Demo Link](https://google.com)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- GETTING STARTED -->
+- [Live Demo Link]( https://mumbereh.github.io/Capston-project/)
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] You have done great so far. :+1: :+1: 
- [ ] As suggested by the last reviewer, the live demo link In your readme doesn't open to your project. Looking at the video below, you can see that the project is not correctly deployed. :+1: You need to correctly deploy your project with Github pages and add the link there. The first issue why this is sharpening could be that you do not have an index.html file. I'll suggest renaming the `home.html` to `index.html`. Remember to update the links also to `index.html` anywhere you called it `home.html`. :smile: :wave: 

- - Video reference:

https://github.com/mumbereh/Capston-project/assets/107632780/3d56f3ee-40e4-4636-aa9c-f260f38f7aae